src/EditorFeatures/Core.Wpf/Suggestions/SuggestedActionsSource.cs

```diff
@@ -200,7 +201,7 @@ public bool TryGetTelemetryId(out Guid telemetryId)
                     var workspace = document.Project.Solution.Workspace;
                     var supportsFeatureService = workspace.Services.GetRequiredService<ITextBufferSupportsFeatureService>();
 
-                    var selectionOpt = TryGetCodeRefactoringSelection(range);
+                    var selection = TryGetCodeRefactoringSelection(range);
```

Comment:
My IDE was telling me to get rid of Opt suffixes; I'm not sure if something I did in this PR is causing this to trigger now.